.github/workflows/ci.yml

```diff
@@ -137,7 +137,7 @@ jobs:
           ./build.php $env:AUTOLOAD
 
       - name: Coveralls
-        run: vendor/bin/php-coveralls -c .coveralls.github-actions.yml -v
+        run: ./vendor/bin/php-coveralls -v --coverage_clover='./dist/tests.clover' --json_path='./dist/coveralls-upload.json'
```

Comment:
do we need service_name too?

Reply:
I actually don't think we do, the command seems to work fine in CI without it?